index.html

```diff
@@ -2595,6 +2595,139 @@ <h3>Data Schemas</h3>
+          <code>resourceIntegrity</code>
+          is present, there MUST be an object in the array for each remote
+          resource for each context used in the verifiable credential. 
+        </p>
+        <p>
+          Each object in the
+          <code>resourceIntegrity</code> array MUST contain the following:
+          the [[URL]] to the resource named <code>id</code> and the 
+          <code>integrity</code> information for the resource
+          constructed using the method specified in <a
+          href="https://www.w3.org/TR/SRI/#integrity-metadata">Subresource
+          Integrity</a>.
+        </p>
+        <p>
+          There MUST NOT be more than one object in the
+          <code>resourceIntegrity</code> per <code>id</code>. 
+        </p>
+        <p>
+          When performing integrity checking the
+          strongest hash algorithm available SHOULD be used to confirm
+          integrity of the resource.
+        </p>
+        <p>
+          An object in the <code>resourceIntegrity</code> array MAY
+          contain a property named <code>timestamp</code>
+          that indicates the time at which the hash was computed. The
+          <code>timestamp</code> property if included MUST be a string
+          value of an [[XMLSCHEMA11-2]] combined date-time string.
+        </p>
+        <p>
+          An object in the <code>resourceIntegrity</code> array MAY contain
+          a property named <code>mediaType</code> that indicates the 
+          expected media type for the indicated <code>resource</code>.
+          If a <code>mediaType</code> is included it must be a valid
+          media type as listed in the 
+          <a href="https://www.iana.org/assignments/media-types/media-types.xhtml">
+            IANA Media Types
+          </a> registry.
+        </p>
+        <p>
+          An object in the <code>resourceIntegrity</code> array MAY contain
+          additional properties.
+        </p>
+        <p>
+          In any object in the <code>credential.credentialSubject</code>
+          that contains a [[URL]], a property named
+          <code>integrity</code> may be included with the integrity
+          information as specified above, with the [[URL]] in the
+          <code>id</code> property. 
+        </p>
+        <p>
+          Any objects for which selective disclosure is desired SHOULD NOT
+          be included as an object in the <code>resourceIntegrity</code>
+          array.
+        </p>
+        <p>
+          Implementers SHOULD consult appropriate sources, such as the 
+          <a href="https://www.iana.org/assignments/named-information/named-information.xhtml">IANA
+          Named Information Hash Algorithm Registry</a> to ensure that they
+          are chosing a current and reliable hash algorithm.  At the time of
+          this writing `sha-256` SHOULD be considered the minimum strength
+          hash algorithm for use by implemnters.
```

Comment:
```suggestion
          are chosing a current and reliable hash algorithm.  At the time of
          this writing `sha-256` SHOULD be considered the minimum strength
          hash algorithm for use by implementers.
```

This goes against the suggestion of the SRI spec (which suggests SHA-384, which we shouldn't suggest). Just flagging this as a deviation from SRI (which I think is fine).

I'll also note that SRI ONLY covers SHA-2, so this property will be stuck w/ SHA-2 forever, which seems problematic given that some government use cases are suggesting SHA-3 now. This is one of the arguments for a more flexible digest expression mechanism, such as digestMultibase, which would allow for other content integrity mechanisms (such as BLAKE-3, KECCAK, SHA-3, canonicalized hashes, and so on).

We should discuss this in the special topic call. Feels like only speaking to SHA-2 is unnecessarily limiting. We don't have to support the full breadth of multihash, but saying that a subset should be supported is probably what we're going for here.

Reply:
this was the reason for the original approach (inline with SRI, but pointing to current hash registry and separating method until SRI is updated) that was removed based on feedback from you and dlongley.  

This increasinging feels like this PR is spinning in circles back to where it started, while also trying to be pulled into something that is not (multibase, etc) - @brentzundel @Sakurann i would request a special topic call on this to resolve.  I also am frustrated by claims that this does not apply to data integrity proofs, as it most certainly does.

Reply:
> this was the reason for the original approach (inline with SRI, but pointing to current hash registry and separating method until SRI is updated) that 

Ok, so you agree that the SHA-2 only limitation is an issue and were looking for a way around it?

> was removed based on feedback from you and dlongley.

The splitting the method from the hash value was requested to be changed because it would result in weird data model statements. Subjects being assigned hash methods, instead of what was intended, which was a process by which you can take a subject URL, hash it, and get an integrity value from it. 

One thing that we could do, that would probably get consensus is to allow this WG to extend the hash mechanisms. We might want to do that for SHA-3 today. So, the format still follows that's in the SRI spec, but allows this WG to add new hash mechanisms as it sees fit.

> This increasinging feels like this PR is spinning in circles back to where it started

It's not, the initial PR wouldn't achieve consensus, but the direction you switched to would. If you end up switching back to the original proposal, I don't expect us to achieve consensus. However, if we keep going down the new path, I expect us to get to consensus.

> while also trying to be pulled into something that is not (multibase, etc)

To be clear, the new path is doing almost exactly what `digestMultibase` does, but using a text-based format. That's why it's coming up, because the approaches are very similar.

> I also am frustrated by claims that this does not apply to data integrity proofs, as it most certainly does.

Can you please elaborate on how using `resourceIntegrity` to protect `@context` values applies to Data Integrity Proofs given that if there is a mismatch between the contexts used to sign and the contexts used to verify that the signatures will not verify (vs. in VC-JWT, they will verify with mismatched contexts)?